pkg/fleet/installer/service/file.go

```diff
@@ -20,6 +20,8 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
+var rollbackNoop = func() error { return nil }
```

Comment:
This is to make sure we always return something when there's no error and avoid accidental panics due to non-nil checking.